<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>861</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
		color: rgb(255, 255, 255)
}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>441</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: rgb(255, 255, 255);
	border-image: url(:/image/images/AcadExcel logo.png);
}
	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>0</y>
       <width>441</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: rgb(76, 111, 191)
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="registBtn">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>111</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border : None;
	background: None;

}</string>
     </property>
     <property name="text">
      <string>Signup</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passL">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>230</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	color:rgb(0, 0, 0)
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>110</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gadugi</family>
      </font>
     </property>
     <property name="text">
      <string>Create New Account?</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userL">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>170</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	color:rgb(0, 0, 0)
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>60</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gadugi</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Account Recovery</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>134</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gadugi</family>
      </font>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>200</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gadugi</family>
      </font>
     </property>
     <property name="text">
      <string>Account Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginBtn">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>410</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gadugi</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color:rgb(121, 149, 212);
	border: None;
}

QPushButton:hover{
	background-color: rgb(255, 255, 255);
	border: None;
	color: rgb(75, 110, 189)
}</string>
     </property>
     <property name="text">
      <string>Recover</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitBtn">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	
border: rgb(0, 0, 0);
background-color:rgb(76, 111, 191);

}

QPushButton:hover{
	background-color: #ff2c2c;
	border: rgb(0, 0, 0);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/images/x icon.png</normaloff>:/image/images/x icon.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="passL_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>290</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	color:rgb(0, 0, 0)
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>260</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gadugi</family>
      </font>
     </property>
     <property name="text">
      <string>New Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passL_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>350</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	color:rgb(0, 0, 0)
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>323</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gadugi</family>
      </font>
     </property>
     <property name="text">
      <string>Repeat New Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bcktolog">
     <property name="geometry">
      <rect>
       <x>685</x>
       <y>111</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border : None;
	background: None;

}</string>
     </property>
     <property name="text">
      <string>Back to Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>110</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gadugi</family>
      </font>
     </property>
     <property name="text">
      <string>or</string>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label</zorder>
    <zorder>registBtn</zorder>
    <zorder>passL</zorder>
    <zorder>label_4</zorder>
    <zorder>userL</zorder>
    <zorder>label_3</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>loginBtn</zorder>
    <zorder>exitBtn</zorder>
    <zorder>passL_2</zorder>
    <zorder>label_7</zorder>
    <zorder>passL_3</zorder>
    <zorder>label_8</zorder>
    <zorder>bcktolog</zorder>
    <zorder>label_9</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
